src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1388,6 +1387,11 @@ public ErrorDiagnostic UnknownModuleReferenceScheme(string badScheme, ImmutableA
                 "BCP234",
                 $"The ARM function \"{armFunctionName}\" failed when invoked on the value [{literalValue}]: {message}");
 
+            public ErrorDiagnostic NoJsonTokenOnPathOrPathInvalid() => new(
+                TextSpan,
+                "BCP235",
+                $"Specified path does not exist in given JSON file or is invalid.");
```

Comment:
`path` could be confused with the file path. What about `JSONPath`?

Reply:
fixed